src/telemetry.js

```diff
@@ -87,12 +90,25 @@ Telemeter.prototype.captureError = function (
   rollbarUUID,
   timestamp,
 ) {
-  var metadata = {
-    message: err.message || String(err),
-  };
+  const message = err.message || String(err);
+  var metadata = {message};
```

Comment:
I think metadata can be `const`. Const prevents reassignment, not mutation.

Reply:
Right. That was an existing line I updated and overlooked the `var`.